parcelgen.py

```diff
@@ -143,8 +143,11 @@ def gen_parcelable_line(self, typ, member):
         elif self.list_type(typ):
             return self.gen_list_parcelable(typ, memberized)
         elif self.map_type(typ):
-            if self.map_type(typ)[1] in self.BOX_TYPES:
+            map_type = self.map_type(typ)
```

Comment:
Is it worth extracting this out into a function??  If we do, it might even be worth writing tests around it (although Idk what the parcelgen deprecation timeline looks like)

Reply:
I don't think it would make sense to extract these into a function, since I'm modifying existing functions to add the functionality of supporting two level maps

Reply:
Note: dennis is the author of the python portion of this change.  

I talked to paulm on core and they are currently working on a tool to take a swagger spec and directly generate a Java model.  Deprecating parcelgen will be part of this process and it seemed that this would happen within 3-6 months(?).